include/seqan3/range/views/minimiser.hpp

```diff
@@ -136,18 +136,9 @@ class minimiser_view : public std::ranges::view_interface<minimiser_view<urng_t>
      *
      * No-throw guarantee.
      */
-    auto end()
```

Comment:
I am confused. Why did you remove the non-const overload completely?

Reply:
You only need  a `const` version if you have a different return type.